src/diffpy/utils/scattering_objects/diffraction_objects.py

```diff
@@ -763,25 +763,29 @@ def q_to_tth(self):
 
             2\theta_n = 2 \arcsin\left(\frac{\lambda q}{4 \pi}\right)
 
+        Function adapted from scikit-beam.  Thanks to those developers
```

Comment:
Pull the line here because I think it's causing issues in the API... which takes it as a parameter.